Xamarin.Forms.Core/Visual.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+
+namespace Xamarin.Forms
+{
+	public static class VisualMarker
+	{
+		public static IVisual MatchParent { get; } = new VisualRendererMarker.MatchParent();
+		public static IVisual Default { get; } = new VisualRendererMarker.Default();
+		public static IVisual Material { get; } = new VisualRendererMarker.Material();
+	}
+
+	public static class VisualRendererMarker
+	{
+		public sealed class Material : IVisual { }
+		public sealed class Default : IVisual { }
+		internal sealed class MatchParent : IVisual { }
+	}
+
+	[TypeConverter(typeof(VisualTypeConverter))]
+	public interface IVisual
+	{
+
+	}
+
+	[Xaml.TypeConversion(typeof(IVisual))]
+	public class VisualTypeConverter : TypeConverter
+	{
+		public override object ConvertFromInvariantString(string value)
+		{
+			if (value != null)
+			{
+				switch (value.Trim().ToLowerInvariant())
```

Comment:
I like the concise switch statement, but I wonder if we should use the same pattern as https://github.com/xamarin/Xamarin.Forms/pull/4501/files#diff-27bd44696d3c1c400c11e4507cababd6R22

Reply:
@StephaneDelcroix  added this code

@StephaneDelcroix  thoughts?

Reply:
ToLowerInvariant is more expensive than Equals with a StringComparison
also, use `nameof()`

I agree with @samhouts , but this is not a blocker

Reply:
changed